enhancements/installer/ibm-cloud-for-power-vs-ipi.md

```diff
@@ -0,0 +1,231 @@
+7. Bootstrapping completes as normal, and workers are added.
+
+### User Stories
+
+- A user of this installer would like to see a cluster running within an hour of kicking off the installer.
+- The expectation is that this is fully automated.
+
+### Implementation Details/Notes/Constraints
+
+#### CoreOS OVA Considerations
+There were two main implementation ideas for making the CoreOS OVA boot image available to the Virtual Server Instance.
+
+1. Import the Image from a Cloud Store
+This is the implementation we believe is best, since it works most similarly to other cloud environments. In this setup,
+the CoreOS build pipeline would publish an OVA image to a cloud store (e.g. AWS bucket). Then, we will use the regular
+IBM Cloud APIs to import the image to the instance. The drawback to this approach is the the current implementation is
+very slow. In its current state, it takes about 30 minutes to import an 120 Gb boot image. This is too long to meet our
+target of being able to install a whole cloud within the hour. One of the reasons this is slow is because we cannot do
+the image resizing after the copy, since it's writing directly to a sized disk. Once imported, creating the nodes is
+fast, since the other disks are flashed from the snapshot. For now, we asked the team to look into meeting the
+requirement that importing the image can take 5 minutes or less.
+
+2. Published CoreOS Images to the IBM Cloud Pre-Authorized Default Images
+This approach would involve outputting our CoreOS OVA images in a queue where they can be pre-loaded into the IBM Cloud
+so that they are available as soon as the Virtual Server Instance is available. This is significantly quicker than the
+method above but has several drawbacks. For one, these default images need to be pre-authorized in order for them to be
+added to the IBM Cloud. This is currently a manual process. Additionally, while we could build automation to solve this,
+it's unclear how would allow the user to point to the correct CoreOS image in the default list, since they would have to
+know the image name and details before their instance is created. Finally, we don't want to muddy up the default image
+list with an ever growing list of CoreOS options. This pruning would need to be automated some other mechanism would
+need to be in place to protect the user experience.
+
+#### DHCP
+The Power VS environment doesn't currently have a way of handling DHCP within the environment. We've asked their team to
+look into how they can provide this, since static IPs are currently not supported for IPI installs.
+
+#### Load Balancing
+When it came to load balancer, the best option we came up with was to use the Direct Link IBM Cloud offering, since it
+allows us to provide an application load balancer to the cluster that can talk directly to the private network. This
+would come at no additional cost to the user, since links are already provided with subscriptions to Power VS.
+
```

Comment:
Added link in latest revision.

Reply:
At the moment, I heard that the IBM Cloud installer uses CIS (instead of IBM Cloud DNS) because it is a pre-requisite.  They will start working on IBM Cloud DNS next.  CIS is for "public DNS" and IBM Cloud DNS is for "private DNS".    Can you address whether Power VS will also have CIS, or are both of these covered via association with the IBM Cloud API?

Reply:
We've got CIS DNS built in to our code-base atm. Thanks @candita for that comment. We should make this clear here.

Reply:
@jaypoulz here are a few links: https://cloud.ibm.com/docs/cis?topic=cis-about-ibm-cloud-internet-services-cis#dns-features, https://cloud.ibm.com/docs/cis?topic=cis-about-ibm-cloud-internet-services-cis.